contracts/test/BoxExtension.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+
+import "./Box.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
+
+contract MockUpgradedVersion {
+    string public constant VERSION = "mock_tag";
+}
+
+contract VeryLongVersionTag {
+    string public constant VERSION =
+        "blahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblah";
+}
+
+/**
+ * Contract adding a variable create a unique contract, that a Box may be upgraded as.
+ */
+contract BoxExtension is BaseBox, MockUpgradedVersion {
+    uint256 private _difference;
```

Comment:
suggestion: remove the unused `uint256 private _difference` from both `BoxExtension` and `BoxExtensionWithVeryLongVersionTag`

That was in the original to show that adding a variable to a contract when upgrading doesn't break things, in this test you're interested in showing the change of the version

Reply:
Done!

Reply:
Strange, it's showing up in the GitHub review diff 🤔